transforms/images/image-assembler-plugin/src/polus/plugins/transforms/images/image_assembler/image_assembler.py

```diff
@@ -0,0 +1,382 @@
+                    Yi = [max(0, y_min - f["posY"])]
+                    Yi.append(min(f["height"], y_max - f["posY"]))
+
+                    # Load the image
+                    with BioReader(f["file"], max_workers=active_threads.count) as br:
+                        image = br[
+                            Yi[0] : Yi[1], Xi[0] : Xi[1], z, 0, 0
+                        ]  # only get the first c,t layer
+
+                    # Put the image in the buffer
+                    template[Yt[0] : Yt[1], Xt[0] : Xt[1]] = image
+
+        # Save the image
+        bw.max_workers = ProcessManager._active_threads
+        bw[y_min:y_max, x_min:x_max, z : z + 1, 0, 0] = template
+
+
+def get_number(s: typing.Any) -> typing.Union[int, typing.Any]:
+    """Check that s is number.
+
+    This function checks to make sure an input value is able to be converted
+    into an integer. If it cannot be converted to an integer, the original
+    value is returned.
+
+    Args:
+        s: An input string or number
+    Returns:
+        Either ``int(s)`` or return the value if s cannot be cast
+    """
+    try:
+        return int(s)
+    except ValueError:
+        return s
+
+
+def _parse_stitch(
+    img_path: pathlib.Path,
+    stitching_vector: pathlib.Path,
+    timepoint_name: bool = False,
+) -> StitchingVector:
+    """Load and parse image stitching vectors.
+
+    This function parses the data from a stitching vector, then extracts the
+    relevant image sizes for each image in the stitching vector to obtain a
+    stitched image size. This function also infers an output file name.
+
+    Args:
+        img_path : path to the images
+        stitching_vector: the stitching vector uses to select the relevant images
+        timepoint_name: Use the vector timeslice as the image name
+    Returns:
+        Dictionary with keys (width, height, name, filePos)
+    """
+    assert stitching_vector.is_file
+    
+    # Initialize the output
+    out_dict: StitchingVector = {
+        "width": int(0),
+        "height": int(0),
+        "name": "",
+        "filePos": [],
+    }
+
+    # NOTE Originally, the filepattern was declared global and thus inherited by forked processes.
+    # This would break on non unix platform where forking process is unavailable (windows) or discouraged (osx)
+    # see [issue](https://github.com/python/cpython/issues/77906)
+    # Alternatively, we could initialized filepattern once and serialize it in each child process but benefits would be unclear.
+    # We could also remove it altogether and select files in the directory using the stitching vector info. Trade-offs are unclear
+    # until some benchmarked are performed.
+
+    # A fail parsing in a subprocess should stop further processing.
+    pattern = ".*"
+    # Try to infer a filepattern from the files on disk for faster matching later
+    try:
+        pattern = filepattern.infer_pattern(img_path)
```

Comment:
@agerardin @Nicholas-Schaub 
I think this is causing the issue in assembler plugin. filepattern.infer_pattern function should infer from list rather than path